app/controllers/airports_controller.rb

```diff
@@ -0,0 +1,5 @@
+class AirportsController < ApplicationController
+  def index
+    
```

Comment:
when creating an empty controller method you should not include a blank line i.e
```
def index
end
```